main.cpp

```diff
@@ -21,6 +22,8 @@ int main(int argc, char *argv[]) {
   a.setWindowIcon(QIcon(":/icon.ico"));
   a.setAttribute(Qt::AA_DisableWindowContextHelpButton);
 
+  initialize_static_path_header_data();

```

Comment:
If this isn't called after a QApplication exists, it will crash trying to construct a QIcon or QPixmap because the icon engine isn't initialized. That's why I couldn't just do a static object initialization.